source/d1amp.cpp

```diff
@@ -2,11 +2,7 @@
 
 #include <QBuffer>
 #include <QDataStream>
-
-D1Amp::D1Amp(QString path)
-{
-    this->load(path);
-}
+#include <QMessageBox>
```

Comment:
I don't think this import is needed now that the dialogs have been moved to `getValidSaveOutput()`